lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -51,6 +55,48 @@ defmodule Trento.Clusters do
     |> Repo.preload([:tags, :hosts_executions, :checks_results])
   end
 
+  @spec get_hosts_connection_settings(String.t()) :: [
+          %{
+            host_id: String.t(),
+            hostname: String.t(),
+            user: String.t()
+          }
+        ]
+  def get_hosts_connection_settings(cluster_id) do
+    query =
+      from h in HostReadModel,
+        left_join: s in HostConnectionSettings,
+        on: h.id == s.id,
+        select: %{host_id: h.id, hostname: h.hostname, user: s.user},
+        where: h.cluster_id == ^cluster_id,
+        order_by: [asc: h.hostname]
+
+    Repo.all(query)
+  end
+
+  @spec save_hosts_connection_settings([
+          %{
+            host_id: String.t(),
+            user: String.t()
+          }
+        ]) :: :ok
+  def save_hosts_connection_settings(settings) do
```

Comment:
I think this should go in the `hosts` usecase in any case

Reply:
Ok, fair enough. As we don't even use the cluster_id to save, and we only use host related data, I see it more in the host use case. But well, it doesn't really care, right? jeje